.storybook/main.js

```diff
@@ -24,16 +24,31 @@ module.exports = {
 
     config.module.rules.push({
       test: /\.scss$/,
-      use: ['style-loader', 'css-loader', 'sass-loader'],
-      include: path.resolve(__dirname, '../'),
+      use: ["style-loader", "css-loader", "sass-loader"],
+      include: path.resolve(__dirname, "../"),
+      exclude: /\.module\.scss$/,
```

Comment:
Excluding `module.scss` files so they won't be processed with these loaders. Since we don't have `modules: true` option here, it will fail to load `module.scss` files.

Keeping it here, because we still need to process `css, scss` files without modules.